src/lite-yt-embed.js

```diff
@@ -98,13 +110,30 @@ class LiteYTEmbed extends HTMLElement {
         LiteYTEmbed.preconnected = true;
     }
 
-    addIframe(){
-        const iframeHTML = `
-<iframe width="560" height="315" frameborder="0"
-  allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen
-  src="https://www.youtube-nocookie.com/embed/${this.videoId}?autoplay=1"
-></iframe>`;
-        this.insertAdjacentHTML('beforeend', iframeHTML);
+    onYouTubeIframeAPIReady() {
+        const videoWrapper = document.createElement('div')
+        videoWrapper.id = this.videoId;
+        document.body.appendChild(videoWrapper);
```

Comment:
Ditto on only be manipulating the custom element structure. Not the main body. Please use 'this.appendChild' or reference a node within the element to bind to.

Reply:
Good point 👍